shopfloor_mobile_base/static/wms/src/main.js

```diff
@@ -29,12 +30,18 @@ var EventHub = new Vue();
 
 Vue.mixin(GlobalMixin);
 
-let app_components = {};
-_.forEach(process_registry.all(), function(process, key) {
-    app_components[process.key] = process.component;
-});
-if (app_components.length)
-    console.log("Registered component:", app_components.join(", "));
+let APP_COMPONENTS = {};
+
+const register_app_components = function(components) {
+    _.forEach(components, function(process, key) {
```

Comment:
why const ? 

and why _.foreach ? is components.foreach not good enough ?

Reply:
no special reason. It was like this somewhere else, I used the same approach. Added to roadmap.